src/config.h

```diff
@@ -44,5 +47,14 @@ class Config {
   CoreConfig core;
   AuthConfig auth;
   DeviceConfig device;
+
+ private:
+  std::string getStringValue(boost::property_tree::ptree pt,
```

Comment:
This is static helper function that doesn't need to be in this class. Let's put in anonymous namespace in the cpp. Need to keep header files clean as possible.